### PR TITLE
[11.0][MIG] crm_deduplicate_acl

### PR DESCRIPTION
**Pull request by pedrobaeza**
@Tecnativa

### CODE REVIEW

File: crm_deduplicate_acl/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.returns('self', lambda value: value.id)
+    def message_post(self, **kwargs):
```
Comment (yajo) [inline crm_deduplicate_acl/models/res_partner.py]: This changes upstream method signature. Use `*args, **kwargs`

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
Lol, I can't believe I'm saying this, but please squash translation commits 😋

**Comment by pedrobaeza** [comment]
Transbot commits can't be squashed together as they are not adjacent.

File: crm_deduplicate_acl/models/res_partner.py (hunk 1 above)
Comment (pedrobaeza) [reply crm_deduplicate_acl/models/res_partner.py]: I copied from upstream, so are you sure?

Comment (yajo) [reply crm_deduplicate_acl/models/res_partner.py]: Yep, see: https://github.com/odoo/odoo/blob/622b1d23706aa5fdebeb0a6a467e0e993355efb6/addons/mail/models/mail_thread.py#L1770-L1772

**Comment by yajo** [comment]
Oh, yes you can, just move all the commits together when doing `git rebase -i` and fixup all into the first

**Comment by pedrobaeza** [comment]
> Oh, yes you can, just move all the commits together when doing git rebase -i and fixup all into the first

It's not a question of handling git. It's a question that there are changes in translations in the middle commits, so conflicts arise.

**Comment by pedrobaeza** [comment]
@Yajo I have been able to squash 2 Transbot commits, but not the rest.

Arguments of method fixed.

**Comment by pedrobaeza** [comment]
@Yajo please re-review

**Comment by pedrobaeza** [comment]
@Yajo @rafaelbn please review

**Review by yajo — APPROVED** [review]
Great! I wonder why was it implemented like that before...

File: crm_deduplicate_acl/README.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+* `Arrow <https://openclipart.org/detail/131875/convergent>`_.
+* `Lock <http://fontawesome.io/icon/unlock-alt/>`_.
+
+Contributors
+------------
+
+* Rafael Blasco <rafabn@antiun.com>
```
Comment (rafaelbn) [inline crm_deduplicate_acl/README.rst:76]: Please change to my new address

**Review by chienandalu — APPROVED** [review]

File: crm_deduplicate_acl/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# Copyright 2016 Tecnativa - Vicent Cubells
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Deduplicate Contacts ACL",
+    "summary": "Contact deduplication with fine-grained permission control",
+    "version": "11.0.1.0.0",
+    "category": "Tools",
+    "website": "http://www.tecnativa.com",
```
Comment (rafaelbn) [inline crm_deduplicate_acl/__manifest__.py]: Shouldn't be OCA?

**Review by rafaelbn — APPROVED** [review]
Some comments but :+1:

**Comment by ernestotejeda** [comment]
This module, in the wizards/partner_merge_view.xml file, tries to add the group named "Manually" to the action named "Merge Selected Contacts" visualized on the top of the "res.partner" tree/form views, but it doesn't work. Please, See Pull Request https://github.com/OCA/crm/pull/199/commits/1ba3c19fbe3e191cc9d044e2d973b01c0b4482e4 where i fix that for version 10 of this module.
Thanks.

File: crm_deduplicate_acl/wizards/partner_merge_view.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+                    <attribute name="groups">crm_deduplicate_acl.group_automatically</attribute>
+                </xpath>
+                <xpath expr="//button[@name='action_update_all_process']" position="attributes">
+                    <attribute name="groups">crm_deduplicate_acl.group_automatically</attribute>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="crm.action_partner_merge" model="ir.actions.act_window">
+            <field name="groups_id">group_manually</field>
```
Comment (yajo) [inline crm_deduplicate_acl/wizards/partner_merge_view.xml]: Indeed it seems he's right. Change this to relational notation.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
@pedrobaeza take a look please

**Comment by pedrobaeza** [comment]
@Yajo's comments attended.

**Review by yajo — APPROVED** [review]
Code OK

**Pull request merged**